Malik705017/tomatoClockForPractice
# 前台登入、註冊、登出功能

## Why, What?
接 auth api, user api 實作前台登入,註冊,登出功能, 主要新增：
1. 畫面右上方登入, 註冊按鈕，點擊後跳出 pop up
2. 如果登入後，畫面右上方會跳出輸入的會員名稱
3. 如果登出後，畫面右上方會變回登入註冊按鈕

## How to test
1. cd `./client`
2. `npm install`, require node version v10
3. `npm run start`
4. run your api service
5. if your api service not in 3000 port, you need to `client/src/scripts/config.js` api path
6. click `註冊` button and enter your information
7. verify your email using `api/auth/verify`
8. click `登入` button and enter your account information

## To do
reconfirm how to deal with error status
